frontend/packages/shared/src/contexts/ServicesContext.tsx

```diff
@@ -40,19 +40,24 @@ const handleError = (
   logout: () => Promise<void>
 ): void => {
   // TODO : log axios errors
+  const errorCode = error?.response?.data?.errorCode;
 
   if (error?.response?.status === 401) {
     logout().then(() => window.location.assign(userLogoutAfterPath()));
     return;
   }
 
+  if (errorCode === 'GT_01') {
+    toast.error(t('handle_merge_conflict.discard_changes_toast_message'));
+    return;
+  }
+
```

Comment:
We don't need to handle API errors manually, as they already are handled by this code:

```
  if (errorCode) {
    const errorMessageKey = `api_errors.${errorCode}`;

    if (i18n.exists(errorMessageKey)) {
      toast.error(t(errorMessageKey), { toastId: errorMessageKey });
      return;
    }
  }
```

If we want to display an error message for GT_01, we can just add an error message called `api_errors.GT_01` in the `nb.json` file.